src/util/promise_utils.js

```diff
@@ -368,7 +381,9 @@ function map_values(obj, func) {
                 new_obj[key] = res;
             })
         ))
-        .return(new_obj);
+        .then(function() {
+            return new_obj;
+        });
```

Comment:
```suggestion
        .then(() => new_obj);
```

Reply:
sure, I will do it for all those entries.